src/components/Plans/PlanPrice/PlanPriceWithoutDiscounts.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { getPlanFee, thousandSeparatorNumber } from '../../../utils';
+import PropTypes from 'prop-types';
+
+export const PlanPriceWithoutDiscounts = ({ selectedPlan, selectedDiscount }) => {
+  const intl = useIntl();
+  const formatedFee = thousandSeparatorNumber(intl.defaultLocale, getPlanFee(selectedPlan));
+  return (
+    <>
+      {selectedDiscount?.discountPercentage ? (
+        <span className="dp-price-old">
+          <span className="dp-price-old-money">US$</span>
+          <span className="dp-price-old-amount">{formatedFee}</span>
+        </span>
+      ) : (
+        <></>
+      )}
+    </>
+  );
```

Comment:
What do you think about returning early and reducing the nesting?

```suggestion
  const intl = useIntl();

  if (!selectedDiscount?.discountPercentage) {
    return (<></>);
  }

  const formatedFee = thousandSeparatorNumber(intl.defaultLocale, getPlanFee(selectedPlan));
  return (
    <span className="dp-price-old">
      <span className="dp-price-old-money">US$</span>
      <span className="dp-price-old-amount">{formatedFee}</span>
    </span>
  );
```